core/logger/beholder.go

```diff
@@ -0,0 +1,44 @@
+package logger
+
+import (
+	"context"
+	"fmt"
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder/pb"
+	"google.golang.org/protobuf/proto"
+	"log"
+)
+
+// beholderCustomMessageLogger is used to emit logs as custom messages through Beholder
+type beholderCustomMessageLogger struct {
```

Comment:
ordinarily if you want to log, you would just log. 

However, we will not have access to metrics through Beholder to start so this wrapper is necessary. 

In addition, ST3 requires log messages to be surfaced through Beholder

Reply:
🤔 Is this needed if we go with option A? IIUC it isn't, correct?

Reply:
That's correct - if we go with option A we don't need the wrapped logger